README.md

```diff
@@ -9,14 +9,39 @@ npm install vies-checker
 ```
 
 ### Usage
+
+#### Require/Import the module
 ```js
 const vies = require('vies-checker');
 
+import { viesChecker } from 'vies-checker';
+```
+
+#### Use the isValid function
+```js
 const COUNTRY_CODE = 'IT';
 const VAT_NUMBER = 'B12345678';
 
-// vies.isValid returns a boolean
-const validVAT = await vies.isValid(COUNTRY_CODE, VAT_NUMBER);
+// vies.isValid returns a boolean. If something is wrong, it will throw an error
+try {
+  const validVAT = await vies.isValid(COUNTRY_CODE, VAT_NUMBER);
+  console.log(`VAT number ${(validVAT ? 'is' : 'is not')} in VIES`);
+} catch (e) {
+	console.error(e);
+}
 
-console.log(`VAT number ${(validVAT ? 'is' : 'is not')} in VIES`);
 ```
+
+### Errors
+These are the errors that might ocurr. Use a try/catch to handle them properly.
+
+| Error name                | Description                                                                                                                              | Throws error |
+|---------------------------|------------------------------------------------------------------------------------------------------------------------------------------|--------------|
+| `INVALID_INPUT`             | The provided CountryCode is invalid or the VAT number is empty                                                                           | No           |
```

Comment:
the `INVALID_INPUT` error does not throw an error, from what I saw below.
It will rather return `false`, exactly as expected.

So maybe we should reword this here, as it currently looks like I would need to handle an `INVALID_INPUT` error myself in  a catch block.

what do you think?

Reply:
Great! Maybe don't consider it as an error